cli/commands/project_sync.go

```diff
@@ -323,3 +351,36 @@ func (cmd *ProjectSync) GetUnisonMinorVersion() string {
 	segments := v.Segments()
 	return fmt.Sprintf("%d.%d", segments[0], segments[1])
 }
+
+// Remove the Unison logfile.
+func (cmd *ProjectSync) RemoveLogFile(logFile string, workingDir string) error {
+	command := exec.Command("rm", "-f", logFile)
+	command.Dir = workingDir
```

Comment:
If `logFile` is a variable with `workingDir` already joined, you dont need to pass in `workingDir` and set it the `command.Dir`

Reply:
My reasoning here is that if it is worth having this broken out as a separate function, and an exported one at that, it makes sense to not make assumptions about the logFile path. In fact, you could argue that this should be checking the logFile path exists.

I don't have strong feelings about folding it back in or helping it to stand alone better. Do you have a preference?

Reply:
I think we can keep it, I did not realize you intended to make that funciton for public consumption.  If so, then don't call it `RemoveLogFile` just call it `RemoveFile` and rename the variable and then things make more sense as a general purpose function.